cypress/e2e/models/administration/jira/jira.ts

```diff
@@ -211,7 +214,7 @@ export class Jira {
         } else {
             expectedStatus = "Connected";
         }
-        // This is horrible, but somehow further code inside `cy.get()` takes OLD values from the object and I need to define separate values and use them to override this problem.
+        // As 'within' uses callback function, object values inside of it may differ from values outside. Saving them separately to avoid this affect.
```

Comment:
This comment is common to all within methods , we can add a general doc in utils folder and add it there if needed.

Reply:
I will do that, but still I would like to have it here now